apps/web/src/app/(private)/datasets-v2/page.tsx

```diff
@@ -0,0 +1,37 @@
+import { DatasetsV2Repository } from '@latitude-data/core/repositories'
+import { TableWithHeader } from '@latitude-data/web-ui'
+import { getCurrentUser } from '$/services/auth/getCurrentUser'
+import { RootDatasetHeader } from '$/app/(private)/datasets-v2/_components/RootHeader'
+import { DatasetsTable } from './_components/DatasetsTable'
+import { env } from '@latitude-data/env'
+
+export default async function DatasetsRoot({
+  searchParams,
+}: {
+  searchParams: Promise<{
+    pageSize: string
+    page?: string
+    modal?: 'new'
+  }>
+}) {
+  const { pageSize, page: pageString, modal } = await searchParams
+  const page = pageString?.toString?.()
+  const { workspace } = await getCurrentUser()
+  const scope = new DatasetsV2Repository(workspace.id)
+  const datasets = await scope.findAllPaginated({
+    page,
+    pageSize: pageSize as string | undefined,
+  })
+  return (
+    <TableWithHeader
+      title='Datasets'
+      actions={
+        <RootDatasetHeader
+          isCloud={env.LATITUDE_CLOUD}
+          openNewDatasetModal={modal === 'new'}
+        />
+      }
+      table={<DatasetsTable datasets={datasets} />}
+    />
+  )
+}
```

Comment:
hmm @andresgutgon should i remove this file?

Reply:
Check main. All `datasets-v2` folder is gone

Reply:
I put all the legacy in this folder but the URLs are the same no matter you're with feature flag for `v2` or not
![image](https://github.com/user-attachments/assets/df70a787-98c7-4917-8480-6cda57ea2b87)